src/components/vacation-requests-table/vacation-requests-table-toolbar/vacation-requests-table-toolbar.tsx

```diff
@@ -132,7 +133,19 @@ const TableToolbar = ({
                 setValue={setFormOpen}
               />
             </ToolbarGridItem>
-          )}
+          }
+          {selectedRowIds?.length === 1 && 
+          rows.find((request: VacationsDataGridRow) => request.id === selectedRowIds[0])?.status === VacationRequestStatuses.APPROVED &&
```

Comment:
In this case we are only disabling the edit button if the vacation is approved, but I believe we also wanted to disable the edit button if the request was declined. The logic in the previous comment should take care of this